.github/workflows/ci_macos_arm64_clang.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: CI - macOS arm64 clang
+
+on:
+  schedule:
+    # Weekday mornings at 09:15 UTC = 01:15 PST (UTC - 8).
+    - cron: "15 9 * * 1-5"
+  workflow_dispatch:
+
+concurrency:
+  # A PR number if a pull request and otherwise the commit hash. This cancels
+  # queued and in-progress runs for the same PR (presubmit) or commit
+  # (postsubmit). The workflow name is prepended to avoid conflicts between
+  # different workflows.
+  group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  macos_arm64_clang:
```

Comment:
Add badges to https://github.com/iree-org/iree/blob/b3ff1ed59e359acef0cf6b9b7725eca7098435f1/docs/website/docs/developers/general/github-actions.md?plain=1#L81-L88 and https://github.com/iree-org/iree/blob/b3ff1ed59e359acef0cf6b9b7725eca7098435f1/README.md?plain=1#L42-L48

That could be in this PR or a follow-up, after the first scheduled job runs

Reply:
I added those to this PR.